rdflib/plugins/parsers/hext.py

```diff
@@ -5,7 +5,7 @@
 """
 import json
 
-from typing import List, Union
+from typing import List, Union, cast
```

Comment:
works fine without cast (will add more suggestions to remove usage)

```suggestion
from typing import List, Union
```